src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,65 @@
+import { PeopleFilters } from '../PeopleFilters/PeopleFilters';
+import { Loader } from '../Loader';
+import { PeopleTable } from '../PeopleTable/PeopleTable';
+import { getPeople } from '../../api';
+import { Person } from '../../types';
+import { useCallback, useEffect, useState } from 'react';
+
+export const PeoplePage = () => {
+  const [error, setError] = useState(false);
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(true);
+
+  const fetchPeopleFromApi = useCallback(async () => {
+    try {
+      const todosFromApi = await getPeople();
```

Comment:
The variable `todosFromApi` is misleading as it suggests the data is related to 'todos'. Consider renaming it to `peopleFromApi` or something similar to better reflect the data it holds.